src/encoded/types/dataset.py

```diff
@@ -328,6 +328,27 @@ class Annotation(FileSet, CalculatedVisualize):
         },
     }
 
+    @calculated_property(schema={
+        "title": "Files",
+        "type": "array",
+        "items": {
+            "type": "string",
+            "linkTo": "File",
+        },
+    })
+    def files(self, request, original_files, status):
```

Comment:
Can we not call this files.  Generic names should be avoided when possible.

Reply:
has to be files for API backwards compatibility.

Reply:
@caseylitton I assume Ben's answers satisfies your code change request, so I will resolve this.